src/components/ChatRoomPage/Chat.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IChat } from '../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { currentUserState } from '../../state/atom';
+import userData from '../../db/userData.json';
+
+const Chat = (chat: IChat) => {
+  const currentUser = useRecoilValue(currentUserState);
```

Comment:
하단에서 `isActive={chat.userId === currentUser.userId}` 요 부분이 계속 반복되니까 전역변수로 빼는게 어떨까용???

```suggestion
  const currentUser = useRecoilValue(currentUserState);
  const [isActive, setIsActive] = useState(true);

  useEffect(()=>{
    // 현재 유저 아이디 달라질 때마다 플래그 재설정하는 로직 w/setIsActive
  }, [currentUserState]);
```

Reply:
저도 이 부분은 상태로 빼서 관리하는게 좋은 것 같아요!